reference/providers/kubernetes-v2/index.md

```diff
@@ -150,6 +150,19 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   If instead Spinnaker is deploying ReplicaSets directly without a Deployment,
   this annotation does the job.
 
+* `strategy.spinnaker.io/recreate`
+
+  As of Spinnaker 1.13, you can force Spinnaker to delete a resource (if it
+  already exists) before creating it again. This is useful for kinds such
+  as [`Job`](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+  which cannot be edited once created, or must be recreated to run again.
```

Comment:
Hyphenate "re-created." The hyphen distinguishes re-creation from the set of avocations we think of as "recreation"&mdash;rock climbing, brewery touring, competitive eating...

Reply:
doi! Thanks